src/Lykke.Job.BlockchainCashoutProcessor.Core/Domain/CrossClient/CrossClientCashoutAggregate.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+
+namespace Lykke.Job.BlockchainCashoutProcessor.Core.Domain
+{
+    public class CrossClientCashoutAggregate
+    {
+        public string Version { get; }
```

Comment:
Many of these properties are not used in cross client cashout and just copy pasted from regular cashout - should be removed